README.md

```diff
@@ -11,6 +11,30 @@ The GBot is integrated into FreeCodeCamp
 Join us in Gitter to discuss!
 https://gitter.im/dcsan/gitterbot
 
+## Installation instructions
+
+### Windows
+
+To run GitterBot, you need [Node.js](https://nodejs.org/). Installing Node.js on Windows is incredibly simple:
+
+ - Visit the Node.js homepage at [nodejs.org](https://nodejs.org/) and click the green 'Install' button.
+ - A binary (msi) file for the latest x64 version of Node.js will automatically be downloaded. If you are running a 32-bit version of Windows, click the Downloads button below the Install button and download the appropriate binary.
+ - Follow the prompts in the installer and restart your computer.
+ - Now that you have Node.js installed, you need the actual app itself.
+ - There are two ways to download Gitterbot: either download the ZIP or clone it using Git.
+ -![Download zip or clone](https://i.imgur.com/ZpiaTa4.png?1 "")
```

Comment:
I don't think we should write up about using a zipfile, as then it will be a nightmare for people to contribute code back.

also i want to keep this text less intimidating. more text is more scary...

if you like you could add a section at the bottom on this? "Confused? just want to play with it?"
in which case just going to the chat room maybe enough...